manual/cops_style.md

```diff
@@ -2658,6 +2658,8 @@ foo != bar
 foo == bar
 !!('foo' =~ /^\w+$/)
 !(foo.class < Numeric) # Checking class hierarchy is allowed
+# Blocks with guard clauses are ignored:
+foo.select { |f| next if f.zero?; f != 1 }
```

Comment:
You'll need to regenerate the documentation to update this.

Reply:
d'oh, done!